File: c files/Largest.c
#include <stdio.h>

int main() {
    int n, index, larger;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int integers[n];
    for (index = 0; index < n; index++) {
        printf("Enter the value for element %d: ", index);
        scanf("%d", &integers[index]);
    }
    larger = integers[0];
    for (index = 1; index < n; index++) {
        int temp = integers[index];
        if (temp > larger) {
            int c = larger;
            larger = temp;
            temp = c;
        }
    }
    printf("The largest number is %d\n", larger);
    return 0;
}

File: py files/Largest.py
print("Enter the number of elements")
n = int(input())
integers = [0] * (n)

for index in range(0, n - 1 + 1, 1):
    print("Enter the value for element " + str(index) + " ")
    integers[index] = int(input())
larger = integers[0]
for index in range(1, n - 1 + 1, 1):
    temp = integers[index]
    if temp > larger:
        c = larger
        larger = temp
        temp = c
print("The largest number is " + str(larger) + "")
